mk/spksrc.service.mk

```diff
@@ -406,7 +406,7 @@ $(STAGING_DIR)/$(DSM_UI_DIR)/config:
 	$(create_target_dir)
 	@echo '{}' | jq --arg name "${DISPLAY_NAME}" \
 		--arg desc "${DESC}" \
-		--arg id "com.synocommunity.packages.${SPK_NAME}" \
+		--arg id "com.synocommunity.${SPK_NAME}" \
```

Comment:
as all the packages with an `src/app/config` file and those with deprecated `installer.sh` file use `com.synocommunity.packages.${SPL_NAME)`, I propose to change it the other way round:

leave `spksrc.service.mk` as it is and modify `spksrc.spk.mk`.